internal/util/util.go

```diff
@@ -90,7 +105,7 @@ func ExportFromReader(input io.Reader, opts define.BuildOutputOption) error {
 		if err != nil {
 			return fmt.Errorf("failed while performing untar at %q: %w", opts.Path, err)
 		}
-	} else {
+	default:
```

Comment:
(Consider using `case theOtherExpectedValue:` here, and failing with an internal error in `default`.)